tests/com/oltpbenchmark/api/AbstractTestWorker.java

```diff
@@ -80,11 +80,11 @@ public void testExecuteWork() throws Exception {
         w.initialize();
         assertFalse(this.conn.isReadOnly());
         for (TransactionType txnType : this.workConf.getTransTypes()) {
-            if (txnType.isSupplemental()) continue;
+//            if (txnType.isSupplemental()) continue;
```

Comment:
It looks like supplemental transaction types were removed.

The only supplemental transactions were:

- SEATS LoadConfig
- AuctionMark LoadConfig, CloseAuctions, ResetDatabase

Reply:
Documented in commit.